pkg/client/client.go

```diff
@@ -663,7 +663,7 @@ func (c *KpmClient) CompileGitPkg(gitOpts *git.CloneOptions, compileOpts *opt.Co
 }
 
 // CompileOciPkg will compile the kcl package from the OCI reference or url.
-func (c *KpmClient) CompileOciPkg(ociSource, version string, opts *opt.CompileOptions) (*kcl.KCLResultList, error) {
+func (c *KpmClient) CompileOciPkg(ociSource, version string, opts *opt.CompileOptions, skipTLSVerify bool) (*kcl.KCLResultList, error) {
```

Comment:
Try to move `skipTLSVerify` to `CompileOptions`. Do not add method parameters directly, which is a public method. This breaks compatibility.